web/html/src/components/formulas/EditGroup.js

```diff
@@ -217,7 +217,9 @@ class EditDictionaryGroup extends React.Component {
         return (
             <div className="form-group" key={element_name}>
                 <label className="col-lg-3 control-label" style={{"color": "green"}}>
-                    {element_name + ":"}
+                    {element_name}
+                    <span className="required-form-field"> *</span>
```

Comment:
Is this reflected in the screenshot attached to the PR description? I can't see it.

Reply:
ah yeah, I reused incorrectly the screenshot from the other PR. I updated it with the combo red+green